.github/CODEOWNERS

```diff
@@ -594,3 +571,21 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 
 # ServiceLabel: %Web Apps %Service Attention
 #/<NotInRepo>/            @AzureAppServiceCLI @antcp
+
+# ######## Management Plane ########
+
+/**/*Management*/                               @allenjzhang @m-nash @markcowl @YalinLi0312 @bquantump @nisha-bhatia
```

Comment:
This seems to match the pattern that you called out in your title, but the trailing `/` is still in place.   Was that intentional?

Reply:
The issue is not with the pattern. It is with the fact that people/teams who don't have write permissions to the repo can't be added as owners in the file.